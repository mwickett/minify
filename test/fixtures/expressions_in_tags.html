<div id='{{ testId }}' class='{{ testClass }}'>{{ testContent }}</div>
