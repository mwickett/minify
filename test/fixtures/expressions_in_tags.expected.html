<div id="thisIsId" class="thisIsClass">thisIsContent</div>
